src/app/Media/VoIPMediaSession.cs

```diff
@@ -206,42 +222,56 @@ private void VideoFormatsNegotiated(List<VideoFormat> videoFormats)
             _videoTestPatternSource?.SetVideoSourceFormat(videoFormat);
         }
 
+        private void TextFormatsNegotiated(List<TextFormat> textFormats)
+        {
+            var textFormat = textFormats.First();
+            logger.LogDebug($"Setting text sink and source format to {textFormat.FormatID} {textFormat.FormatName}. (RTP clock rate {textFormat.ClockRate}).");

```

Comment:
As per #1306, use structured semantic logging instead of dynamic log message templates.